pytorch_transformers/tokenization_utils.py

```diff
@@ -666,35 +696,146 @@ def _convert_token_to_id(self, token):
     def encode(self, text, text_pair=None, add_special_tokens=False, **kwargs):
         """
         Converts a string in a sequence of ids (integer), using the tokenizer and vocabulary.
-        
+
         Same as doing ``self.convert_tokens_to_ids(self.tokenize(text))``.
 
         Args:
             text: The first sequence to be encoded.
```

Comment:
We should explain clearly the various possible inputs of the method in the docstring